docs/release-notes.md

```diff
@@ -11,135 +11,117 @@
 
-:material-new-box:** Feature: Add CIRA support for static IP devices**
+When customers use their own authentication server, there should be a way to disable default MPS JWT User Authentication. The configuration setting "web_auth_enabled" in MPS lets users to enable/disable default MPS JWT User Authentication. This configuration is enabled by default and recommend to set it to false only when using a different authentication server. This setting will not affect the new API Endpoint for Short Lived JWTs that is required for Redirection sessions.
 
-With this release we added support for CIRA configurations with static IP addresses. AMT will now connect to an MPS when it is configured with a static IP address.
+:material-new-box:** Improvement: Set minimum TLS version for CIRA connections**
 
-:material-new-box:** Feature: MQTT events for redirection start and stop **
+Some versions of TLS encryption algorithms supported by AMT are weaker than others. This configuration option lets users to enforce minimum TLS version to restrict CIRA connections (connection betwween MPS and AMT) from older versions of AMT. By default, `minVersion` property (within `mps_tls_config`) in `.mpsrc` is set to TLSv1 as the older versions of AMT (i.e., AMT version <=10) use TLSv1. Changing this value to newer versions of TLS will effectively prevent older versions of AMT from connecting.
 
-When redirection sessions start or stop (KVM or SOL), MPS will now send an event to the MQTT broker
+:material-new-box:** Improvement: Removed auto-load toggle and always auto-load certificate**
 
-:material-auto-fix:** Improvement: UI Toolkit Angular - v13 support**
+We removed the auto-load toggle from the CIRA Config page on our Sample Web UI as it is typically used with the full Open AMT Cloud Toolkit. By default, the CIRA certificate will always auto-load so that the UI is more simplified. If you wish to manually provide the CIRA certificate, you may do so using the RPS API directly.
 
-With this release, UI-Toolkit-Angular supports Angular v13. This breaking change is indicated by the major version transition to 3.x.x. If you are still on Angular 12, stick with UI-Toolkit-Angular 2.0.5 until you migrate Angular v13.
+:material-star:** Coming Soon: Set AMT Features with AMT Profile**
 
-:material-auto-fix:** Improvement: UI Toolkit - KVM mouse alignment when scrolled**
+In our next release, we will be supporting the ability to set AMT Features as part of the AMT Profile during activation. This includes settings such as IDE Redirection, and User Consent Mode for KVM.  This feature will eliminate the need for setting these features post-activation.  While we wanted to complete this feature for this release, we weren't quite able to get everything in. However, the database changes to support it are included with this release. If you are migrating from v2.3.0 version of the toolkit, you'll need to add the following columns to the `profiles` table in the RPS Database.
```

Comment:
In our next release, we will be support the ability to set AMT Features as part of the AMT Profile during activation. This includes settings such as IDE Redirection and User Consent Mode for KVM.  This feature eliminates the need for setting these features post-activation. However, the database changes to support this feature are included with this release. If you are migrating from v2.3.0 version of the toolkit, you'll need to add the following columns to the `profiles` table in the RPS Database:

Reply:
done